lib/eventsource.js

```diff
@@ -211,7 +211,28 @@ function EventSource (url, eventSourceInitDict) {
 
       // Handle HTTP errors
       if (res.statusCode !== 200) {
-        failed({ status: res.statusCode, message: res.statusMessage })
+        var chunks = []
+        var returnError = function () {
+          res.removeAllListeners('close')
```

Comment:
Can we log the error?  This would help with investigations.

Reply:
We'll log it when we catch the error that gets emitted here.